docs/SIL.rst

```diff
@@ -5707,6 +5707,49 @@ Extracts the original function or a transpose function from the given
 ``[original]`` or ``[transpose]``.
 
 
+differentiability_witness_function
+``````````````````````````````````
+
+::
+
+  sil-instruction ::= 'differentiability_witness_function'
+                      sil-differentiability-witness-function-kind
+                      '[' 'parameters' sil-differentiability-witness-indices ']'
+                      '[' 'results' sil-differentiability-witness-indices ']'
+                      generic-parameter-clause?
+                      sil-function-name ':' sil-type
+
+  sil-differentiability-witness-function-kind ::=
```

Comment:
Remove this non-terminal.

Reply:
Done in https://github.com/apple/swift/pull/27719/commits/e2335aa94efa1504b9e4385c71d861a34b9348d3. Done for other autodiff instructions in https://github.com/apple/swift/pull/27719/commits/0db4e274d61a91fc8a9872994efd566616a29f62.